Display the previous save as a diff

kaiyue0329 opened the pull request:
#### What it does
Completes: #7 

- User can now view a local history save as a diff compared to the current active editor and clearly identify the differences between two revisions.

#### How to test
- Press F5 to run the tests in a new window with your extension loaded.
- Go to `File` on top menu and select `Add Folder to Workspace`
- Select a workspace folder
- Create a new file inside the `src` folder
- Open the Command Palette (Ctrl+Shift+P), enter `Local History: Local History: Active Editor`
- Select a local history file to view the diff

Signed-off-by: Kaiyue Pan <kaiyue.pan@ericsson.com>

vince-fugnitto commented on src/local-history.ts:
```diff
@@ -138,4 +138,14 @@ export class LocalHistoryProvider {
+    /**
+     * Compare the two revisions and show the diff between them
+     */
+    private viewDiffBtwRevisions(currentVersion: vscode.Uri, historyVersion: vscode.Uri) {
```
[minor] rename `viewDiffBtwRevisions` to `displayDiff(current, previous)`.
[minor] please include a return type (check if we can add a rule for it)

kaiyue0329 commented:
> @kaiyue0329 which do you think is better?
> previous < > current
> current < > previous

I think `previous < > current` is better since this is how git does it.

> For example, I added the above line in the current change and it seems odd to see it as a deletion.

I updated the function. The added line in the current change can now display as an addition.
![image](https://user-images.githubusercontent.com/23107734/79499022-4e454180-7ff8-11ea-8e99-dc588281575b.png)
